packages/common/src/controllers/AppController.ts

```diff
@@ -94,6 +96,11 @@ export default class AppController {
       await getModule(FavoritesController).initialize();
     }
 
+    // useAccessBridge is enabled when the APP_API_ACCESS_BRIDGE_URL environment variable is configured
+    if (useAccessBridge) {
```

Comment:
Was wondering if instead of doing this in multiple places, to conditionally bind two controllers (create one dummy with mocked methods that do nothing). Wdyt @dbudzins ?